post/zzz_copy_sut_archive_files.rb

```diff
@@ -0,0 +1,25 @@
+require 'fileutils'
+require 'time'
+
+step "Copy archive files from SUT" do
+  now = Time.now.getutc.to_i
+  # truncate the job name so it only has the name-y part and no parameters
+  if ENV['JOB_NAME']
+    job_name = ENV['JOB_NAME']
+                   .sub(/[A-Z0-9_]+=.*$/, '')
+                   .gsub(/[\/,.]/, '_')[0..200]
+  else
+    job_name = 'unknown_or_dev_job'
+  end
+
+  archive_name = "#{job_name}__#{ENV['BUILD_ID']}__#{now}__perf-files.tgz"
```

Comment:
build id would only be there for a jenkins job right?   And since archive root has #{now} does the file name need it as well?  Does it need build or jobname either?  Maybe a meta data file instead with any useful information about the source of the data should be added to the archive, instead of encoding it in a file name that is already in a unique directory.  (though if the tgz is all we are going to upload then you would need at least the "now" to keep it unique.

Reply:
Correct, build id will only be there for a jenkins job.  The file name is the name of the archive file that gets created when it zips up the archive directory, it won't be under the folder itself, it will be in the working directory so it does also need to be unique. The reason we want the other info there is because we'll be downloading these from jenkins and they'll show up in our downloads directory, and this is a simple way to keep them organized without having to unzip the archive and open it to see what it was an archive of.

Reply:
But if not in jenkins buildID will be undefined.

Reply:
Correct, when it is run locally it will look like this:
unknown_or_dev_job____1518473738__perf-files.tgz
That's fine, since when we run it locally we aren't having to worry about having it organized by build, it will just be sorted by run time, so the most recent run will be at the bottom.

Reply:
i'm personally fine with this for the time being.  it woulda been cool to replace `nil` with `'local'` or something.  but we don't know how long this will be around for.  so i'm okay with it like this and we can fix it later if it's annoying/confusing.